RELEASE-NOTES.txt

```diff
@@ -6,6 +6,7 @@ Unreleased
 ------
 * [*] Image block: "Set as featured" button within image block settings. (Android only). [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3116]
 * [***] Audio block now available on WP.com sites on the free plan. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3523]
+* [**] Fixed an issue where pressing enter inside a text-based block was not creating a new block when using Gboard [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3590]
```

Comment:
I believe this new note should be in the above "Unreleased" section if I am not mistaken. It currently is listed under the `1.55.0` release, which I don't believe included this fix.

Reply:
Oh good catch! this PR had hung around for a while and I didn't think about it. Fixed in 06eee7b2aaa09100ac9c3afd035b7a37f750d581

Reply:
Thanks for wrangling this!